BLE_BatteryLevel/.mbedignore

```diff
@@ -0,0 +1 @@
+deprecated/*
```

Comment:
For the deprecation pattern I would like to see the old code being compiled on NRF51 and NRF51 only. 

So, depending on the target we would compile one main or another.